src/site/includes/debug.drupal.liquid

```diff
@@ -1,4 +1,10 @@
-{% if buildtype != 'vagovprod' %}
+{% if isPreview or buildtype != 'vagovprod' %}
+  {% if omitdebug == false %}
+    {% assign showDebug = true %}
+  {% endif %}
+{% endif %}
+
+{% if showDebug %}
```

Comment:
```suggestion
{% assign showDebug = false %}

{% if buildtype == 'vagovprod' or omitdebug %}
    {% assign showDebug = false %}
{% endif %}

{% if showDebug %}
```

Reply:
Man, confusing set of booleans here. Liquid scoping is sometimes odd though so I'm wondering about that `showDebug`. Does this work locally for you? `yarn preview` -> navigate to `http://localhost:3001/preview?nodeId=8296` -> check for console output

Reply:
It does not